### PR TITLE
fix: 0.11 deprecations

### PR DESCRIPTION
**Pull request by tris203**
this is kind of pointless until plenary fix upstream too

### CODE REVIEW

File: lua/cokeline/config.lua [hunk 1]
```diff
@@ -9,7 +9,13 @@ local hlgroups = lazy("cokeline.hlgroups")
 local insert = table.insert
 
 local echo = vim.api.nvim_echo
-local islist = vim.tbl_islist
+local islist = function()
```
Comment (willothy) [inline lua/cokeline/config.lua:12]: Hmm, maybe we could just do this check once, like `islist = vim.islist or vim.tbl_islist`?

**Comment by willothy** [comment]
Thanks for the PR!

File: lua/cokeline/config.lua (hunk 1 above)
Comment (tris203) [reply lua/cokeline/config.lua]: yeah im happy with that

**Review by willothy — APPROVED** [review]
Awesome, thank you!

**Pull request merged**